src/legacy/status_im/network/net_info.cljs

```diff
@@ -14,9 +13,7 @@
             {:db (assoc db :network-status (if is-connected? :online :offline))}
             (when (and is-connected?
                        (or (not= (count (get-in db [:wallet-legacy :accounts]))
-                                 (count (get db :profile/wallet-accounts)))
-                           (wallet/has-empty-balances? db)))
-              (wallet/update-balances nil nil))))
+                                 (count (get db :profile/wallet-accounts))))))))
```

Comment:
I guess this whole `when` is not needed at this moment as we don't store data in `:wallet-legacy` or `:profile/wallet-accounts`.

Reply:
true!